docs/sdks/server-sdks/dotnet/bandits.mdx

```diff
@@ -0,0 +1,209 @@
+| `Action` (string)                           | The action assigned by the bandit                                                                                 | "promo-20%-off"                  |
+| `ActionNumericAttributes` (IDictionary)     | Metadata about numeric attributes of the assigned action. Map of the name of attributes their provided values     | `{"brandAffinity": 0.2}`         |
+| `ActionCategoricalAttributes` (IDictionary) | Metadata about non-numeric attributes of the assigned action. Map of the name of attributes their provided values | `{"previouslyPurchased": false}` |
+| `ActionProbability` (double)                | The weight between 0 and 1 the bandit valued the assigned action                                                  | 0.25                             |
+| `OptimalityGap` (double)                    | The difference between the score of the selected action and the highest-scored action                             | 456                              |
+| `ModelVersion` (string)                     | Unique identifier for the version (iteration) of the bandit parameters used to determine the action probability   | "v123"                           |
+| `MetaData` (IDictionary)                    | Any additional freeform meta data, such as the version of the SDK                                                 | `{ "sdkLibVersion": "3.5.1" }`   |
+
+### Query the bandit for an action
+
+To query the bandit for an action, use the `GetBanditAction()` method. The most specific implementation takes the following parameters:
+- `flagKey` (string): The key of the feature flag corresponding to the bandit
+- `subjectKey` (string): The key of the subject or user assigned to the experiment variation
+- `subjectAttributes` (IDictionary): The subject's attributes
+- `actions` (IDictionary): Map of actions (by name) to their categorical and numeric attributes
+- `defaultValue` (string): The default *variation* to return if the flag is not successfully evaluated
+
+```csharp
+var subjectAttributes = new Dictionary<string, object?>()
+{
+    ["age"] = 30,
+    ["country"] = "uk",
+    ["pricingTier"] = "1"
+};
+
+var actions = new Dictionary<string, IDictionary<string, object?>>()
+{
+    ["nike"] = new Dictionary<string, object?>()
+    {
+        ["brandLoyalty"] = 0.4,
+        ["from"] = "usa"
+    },
+    ["adidas"] = new Dictionary<string, object?>()
+    {
+        ["brandLoyalty"] = 2,
+        ["from"] = "germany"
+    }
+};
+
+var result = eppoClient.GetBanditAction(
+    "flag-with-shoe-bandit",
+    "user123",
+    subjectAttributes,
+    actions,
+    "default");
+
+if (result.Action != null)
+{
+    // Follow the Bandit action
+    RenderShoeAd(result.Action);
+} else {
+    // User was not selected for a Bandit.
+    // A variation is still assigned.
+    RenderDefaultShoeAd(result.Variation);
+}
+```
+
+### `GetBanditAction` Overloads
+
+There are several overloads of the `GetBanditAction()` method available:
```

Comment:
Can link to the [source code](https://github.com/Eppo-exp/dot-net-server-sdk/blob/89f65bb5779a7293beff878534381dcc871b03cd/dot-net-sdk/EppoClient.cs#L267) for full examples of using the convenient overloads

Reply:
Good suggestion. Link added.